published/external/afxdp.h

```diff
@@ -16,13 +16,19 @@
 extern "C" {
 #endif
 
-#define XSK_BUFFER_DESCRIPTOR_ADDR_OFFSET_MAX 65535ull
-#define XSK_BUFFER_DESCRIPTOR_ADDR_OFFSET_SHIFT 48
-#define XSK_BUFFER_DESCRIPTOR_ADDR_OFFSET_MASK \
-    (XSK_BUFFER_DESCRIPTOR_ADDR_OFFSET_MAX << XSK_BUFFER_DESCRIPTOR_ADDR_OFFSET_SHIFT)
+typedef union _XSK_BUFFER_ADDRESS {
+    struct {
+        UINT64 BaseAddress : 48;
+        UINT64 Offset : 16;
+#pragma warning(suppress:4201) // nonstandard extension used: nameless struct/union
+    } DUMMYUNIONNAME;
```

Comment:
Why do you need this? We don't use this in MsQuic and it works cross platform, and user and kernel.

Reply:
Unnamed unions violate the C spec. Quite possibly all the MsQuic compilers support this extension.

Reply:
Perhaps, but it works with every compiler and platform I've seen, which has been quite extensive. And XDP runs on only a subset of those.

Reply:
That's fair. I can remove it if you feel strongly we shouldn't litter our code with this.

Reply:
My personal preference is to remove this, but I leave it up to you.